third_party/OWNERS

```diff
@@ -4,12 +4,7 @@
 {
   rules: [
     {
-      owners: [
-        {name: 'rsimha'},
-        {name: 'erwinmombay'},
-        {name: 'choumx'},
-        {name: 'jridgewell'},
-      ],
+      owners: [{name: 'erwinmombay'}, {name: 'choumx'}, {name: 'jridgewell'}],
```

Comment:
Maybe remove `choumx` while you're here?

Reply:
Codesearch shows dozens of instances where owners files contain folks who have left the team. I'll leave this for a separate cleanup.